requirements.txt

```diff
@@ -5,3 +5,4 @@ mock==1.0.1
 testfixtures==4.1.2
 paramiko
 git+ssh://git@github.com/ministryofjustice/bootstrap-cfn.git@v0.3.0#egg=bootstrap-cfn>=0.3.0
+requests
```

Comment:
This needs to be added to the setup.py too as this file is ignored when installing bootstrap-salt via pip